applications/fov-quicklook/templates/secret.yaml

```diff
@@ -0,0 +1,39 @@
+{{- if .Values.use_vault }}
+apiVersion: ricoberger.de/v1alpha1
+kind: VaultSecret
+metadata:
+  name: fov-quicklook-secret
+spec:
+  path: "{{ .Values.global.vaultSecretsPathPrefix }}/fov-quicklook"
+  type: Opaque
+  keys:
+    - s3_repository_acceess_key
```

Comment:
```suggestion
    - s3_repository_access_key
```

Reply:
fixed in f0d243d

Reply:
fixed in 137df11551e17460ef6b3dcba721a23940aef9de